app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ActiveRecord::Base
+  validates :user_name, presence: true, uniqueness: true
+  validates :password, presence: true
+  has_many :jobs
+  has_secure_password
+  enum role: %w(default admin)
+
+  def admin?
```

Comment:
this method is already given to you via line 6 - this is one of the benefits of using an enum :thumbsup: